conans/test/functional/build_helpers/cmake_flags_test.py

```diff
@@ -474,8 +474,8 @@ def package(self):
         client.run("create . danimtb/testing")
         if platform.system() == "Windows":
             self.assertIn("WARN: CMake generator could not be deduced from settings", client.out)
-            self.assertIn('Configure command: -DCONAN_EXPORTED="1" -DCONAN_IN_LOCAL_CACHE="ON" '
+            self.assertIn('Configure command: -DCONAN_IN_LOCAL_CACHE="ON" '
```

Comment:
I guess the ``-DCONAN_EXPORTED=1`` is printed, but later. Could you please add that check?

Reply:
Yes, it is, but I think that this test is checking that the generator is printed for "Unix Makefiles", but not for Windows. The test was added in https://github.com/conan-io/conan/pull/4509 ("Revert CMake default generator on Windows"), nothing related to `CONAN_EXPORTED`.

Reply:
good then, can be left as-is and merged.